polling_stations/apps/data_finder/helpers/every_election.py

```diff
@@ -287,6 +287,18 @@ def multiple_elections(self):
             return len(uncancelled_ballots) > 1
         return False
 
+    @property
+    def has_city_of_london_ballots(self):
+        # City of London local elections have some edge cases e.g:
+        # diffrent polling station opening times, different registration rules
+        if not self.request_success:
+            return False
+        uncancelled_ballots = [b for b in self.ballots if not b["cancelled"]]
+        for b in uncancelled_ballots:
+            if b["election_id"].startswith("local.city-of-london"):
```

Comment:
This will catch elections for `https://elections.democracyclub.org.uk/organisations/local-authority/LND-alder/1189-01-01/` and `https://elections.democracyclub.org.uk/organisations/local-authority/LND/1189-01-01/` I assumed this is desired, but thought it worth double checking

Reply:
Yes. Both Alderman and Common Council have the same rules here.